src/test/eo/org/eolang/collections/hash-code-of-tests.eo

```diff
@@ -0,0 +1,68 @@
+# The MIT License (MIT)
+#
+# Copyright (c) 2021-2022 Yegor Bugayenko
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included
+# in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
++alias org.eolang.collections.hash-code-of
++alias org.eolang.hamcrest.assert-that
++alias org.eolang.math.number
++home https://github.com/objectionary/eo-collections
++junit
++package org.eolang.collections
++version 0.0.0
+
+[] > hash-code-bools-is-integer
+  assert-that > @
+    and.
+      is-int.
+        number
+          hash-code-of TRUE
+      is-int.
+        number
+          hash-code-of FALSE
+    $.equal-to TRUE
+
+[] > hash-codes-of-the-same-numbers-are-equal
+  42 > forty-two!
+  assert-that > @
+    hash-code-of forty-two
+    $.equal-to
+      hash-code-of forty-two
+
+[] > hash-code-of-the-same-strings-are-equal
+  "hello" > str!
+  assert-that > @
+    hash-code-of str
+    $.equal-to
+      hash-code-of str
+
+[] > hash-code-of-different-numbers-are-different
+  assert-that > @
+    hash-code-of 42
+    $.not
+      $.equal-to
+        hash-code-of 24
+
+[] > hash-of-int-is-int
```

Comment:
@maxonfjvipon Please, move that test near to the `hash-code-bools-is-integer`

Reply:
Done